README.md

```diff
@@ -1,75 +1,99 @@
+
+## I built these 2 pages:
+
+- The home page.
+- The about page.
+
+## Each of these pages should have versions for 2 different screen sizes:
+
+- **Mobile**: up to 768px wide.
+- **Desktop**: 768px or wider.
+
+## Interactions:
+
+- The home page has a link in the menu to the about page.
+- The logo in the header links to the home page.
+- **Mobile menu**
+   - When the user clicks the hamburger button on the header, the mobile menu appears over the page.
+   - The mobile menu has a close (X) button that closes the menu.
+- **Dynamic page**
+   - The section "Featured speakers" should be created dynamically in JavaScript.
+   - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
+ 
+> Original design idea by Cindy Shin in Behance.
+
+## video presentation
+
+[Leboroz video](https://www.loom.com/share/2bd2aad2302a4081a84cb0192dae71ed)
 
 ## Built With
 
 - Major languages
-- Frameworks
+    - HTML5
+    - SASS
+    - Javascript
 - Technologies used
+    - npm
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://leboroz.github.io/capstone-project-1)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**project doesn't require testing.**
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+    - npm
+    - git
 
 ### Setup
 
-### Install
+open a terminal and run ```git clone git@github.com:Leboroz/capstone-project-1.git```
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
+opening the html file will let you see the live version.
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## Author
 
-👤 **Author2**
+👤 **Leonardo Albornoz**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@leboroz](https://github.com/leboroz)
+- Twitter: [@leboroz](https://twitter.com/leboroz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Leonardo_albornoz)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/leboroz/capstone-project-1/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- GitHub: [@RafaelEchart](https://github.com/RafaelEchart)
 
 ## 📝 License
```

Comment:
- [ ] The README should contain an "Acknowledgements" which gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons](https://creativecommons.org/licenses/) License. Please consider adding it to your acknowledgment section of the README. 😄